gremlin-server/pom.xml

```diff
@@ -148,6 +149,29 @@ limitations under the License.
     </build>
     <profiles>
         <profile>
+            <id>linux</id>
+            <activation>
+                <os>
+                    <family>Linux</family>
```

Comment:
Doesn't this mean that if I'm on linux, i'll always be testing epoll as the profile "activates" when the os is linux?  I think we'd want the flexibility to be able to test java nio on linux, which we don't get on this approach.  In other words, instead of activating by os, it should activate when requested with:

```xml
<activation>
  <activeByDefault>false</activeByDefault>
  <property>
    <name>epoll</name>
  </property>
</activation>
```